### PR TITLE
Add quality field to transcodes

### PR DESCRIPTION
**Pull request by seb-b**

### CODE REVIEW

File: wagtailvideos/models.py [hunk 1]
```diff
@@ -31,11 +31,39 @@
...
+
+
 class MediaFormats(ChoiceEnum):
     webm = 'VP8 and Vorbis in WebM'
     mp4 = 'H.264 and MP3 in Mp4'
     ogg = 'Theora and Voris in Ogg'
 
+    def get_quality_param(self, quality):
+        if self is MediaFormats.webm:
+            if quality is VideoQuality.lowest:
```
Comment (timheap) [inline wagtailvideos/models.py]: Enums can be used as dict keys, so you could do something like:

```
return {
    VideoQuality.lowest: '50',
    VideoQuality.default: '22',
    VideoQuality.highest: '4',
}[quality]
```

although now that I've typed it out, it is actually no shorter at all...

File: wagtailvideos/templates/wagtailvideos/videos/edit.html [hunk 2]
```diff
@@ -52,13 +52,13 @@
             <h2 class="label">{% trans "Video preview" %}</h2>
 			{% video video controls style=max-width:100% %}
 			<h3 class="label">Transcodes</h3>
-			<p>If you wish to generate HTML5 compliant transcodes use the form below. This may take a while dependung in the length of the video.</p>
+			<p>If you wish to generate HTML5 compliant transcodes use the form below. This may take a while depending in the length of the video.</p>
```
Comment (timheap) [inline wagtailvideos/templates/wagtailvideos/videos/edit.html]: > depending *on* the length

File: wagtailvideos/templates/wagtailvideos/videos/edit.html [hunk 3]
```diff
@@ -75,6 +75,7 @@ <h3 class="label">Create transcode</h3>
 				<ul class="fields">
 					{% csrf_token %}
 					{% include "wagtailadmin/shared/field_as_li.html" with field=transcode_form.media_format %}
+                    {% include "wagtailadmin/shared/field_as_li.html" with field=transcode_form.quality %}
```
Comment (timheap) [inline wagtailvideos/templates/wagtailvideos/videos/edit.html]: Mixed tabs and spaced! The shame...

File: wagtailvideos/models.py (hunk 1 above)
Comment (seb-b) [inline wagtailvideos/models.py]: I like it its cleaner. Anything to get around if/else/if/else..

**Pull request merged**